tests/ut/codegen/test_distributed_codegen.py

```
@@ -710,5 +711,38 @@ def worker(x: pl.Tensor[[64], pl.FP32]):
+
+        @pl.program
+        class Input:
+            @pl.function(type=pl.FunctionType.InCore)
+            def down_seed(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                y: pl.Tensor[[64], pl.FP32] = pl.add(x, x)
+                return y
+
+            @pl.function(type=pl.FunctionType.Orchestration)
+            def decode_fwd(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                with pl.manual_scope():
+                    a, a_tid = pl.submit(self.down_seed, x)
```

Ruff (0.15.15)

[warning] Unpacked variable `a_tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)